src/Mealz/MealBundle/Service/ParticipationService.php

```diff
@@ -65,6 +68,21 @@ public function join(Profile $profile, Meal $meal, ?Slot $slot = null, array $di
         return null;
     }
 
+    /**
+     * Update combined meal dishes for a participant.
+     *
+     * @param string[] $dishSlugs
+     *
+     * @throws ParticipationException
+     */
+    public function update(Participant $participant, array $dishSlugs): void
```

Comment:
A generic name 'update' for a very special use case: updateCombinedDishes.

Reply:
This is actually participant update. The controller as well as the entity are named after participant.

Reply:
Ultimately changed it to what you suggested. :)